06_gpu_and_ml/whisper_pod_transcriber/main.py

```diff
@@ -411,16 +410,9 @@ def fetch_episodes(show_name: str, podcast_id: str, max_episodes=100):
     return episodes
 
 
-if __name__ == "__main__":
-    cmd = sys.argv[1]
-    if cmd == "serve":
-        stub.serve()
-    elif cmd == "index":
-        with stub.run():
-            index()
-    elif cmd == "search-podcast":
-        with stub.run():
-            for pod in search_podcast.call(sys.argv[2]):
-                print(pod)
-    else:
-        exit(f"Unknown command {cmd}. Supported commands: [transcribe, run, serve, index, search-podcast]")
+@stub.local_entrypoint
+def search_entrypoint(name):
```

Comment:
```suggestion
def search_entrypoint(name: str):
```

Would work anyway, but good habit to have the types as they determine arg parsing.

Reply:
yeah good point, thanks!